core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -309,6 +309,19 @@ object ErrorSource {
   def stringToObject: Map[String, ErrorSource] = values.map(v => v.toString -> v).toMap
 }
 
+sealed abstract class AllowedChartName extends Product with Serializable {
```

Comment:
I understand the bigger picture, but I think it would be more prudent to isolate this to AOU apps for now, and rename to `AOUAllowed` for the chart names and AppType.
Also the code below is always referencing `aou*` so it will be easier to be consistent throughout

That will also help us make the AOU app upgrade logic explicit (we will need to modify our code to loop through each chart within the app type for AOU moving forward)

Reply:
had offline discussion about naming...

ALLOWED is meant to be generic in a sense that it's a category of apps doesn't need to be specific to AoU

Will check metrics usage